types.ts

```diff
@@ -492,33 +547,14 @@ export interface OAuth2Authentication extends BaseAuthentication {
    * they may be specified using {@link additionalParams}.
    */
   authorizationUrl: string;
-  /**
-   * The URL that Coda will hit in order to exchange the temporary code for an access token
-   * at the end of the OAuth handshake flow.
-   */
-  tokenUrl: string;
-  /**
-   * Scopes that are required to use this pack.
-   *
-   * Each API defines its own list of scopes, or none at all. You should consult
-   * the documentation for the API you are connecting to.
-   */
-  scopes?: string[];
   /**
    * The delimiter to use when joining {@link scopes} when generating authorization URLs.
    *
    * The OAuth2 standard is to use spaces to delimit scopes, and Coda will do that by default.
    * If the API you are using requires a different delimiter, say a comma, specify it here.
    */
   scopeDelimiter?: ' ' | ',' | ';';
```

Comment:
Any reason not to include this one if we're including `scopes`?

Reply:
Oh ya, good catch!

Reply:
Adding this to the next PR